src/components/MapView/AlertForm/index.tsx

```diff
@@ -0,0 +1,354 @@
+                    setSelectedThreshold(parseInt(e.target.value, 10))
+                  }
+                />
+              </div>
+              <div className={classes.AlertFormOptions}>
+                <Select
+                  id="regionsList"
+                  className="regionsListClass"
+                  label="Regions to monitor"
+                  type="text"
+                  variant="filled"
+                  value={regionsList}
+                  multiple
+                  style={{ maxWidth: '200px' }}
+                  onChange={e => {
+                    const selected: string[] = e.target.value as string[];
+                    const lastSelected = selected[selected.length - 1];
+
+                    if (lastSelected !== 'allRegions') {
+                      setRegionsList(
+                        selected.filter(el => el !== 'allRegions'),
+                      );
+                    } else {
+                      // If 'allRegions' was the last selected then it should be the only thing selected.
+                      setRegionsList(['allRegions']);
+                    }
+                  }}
+                >
+                  [
+                  <MenuItem
+                    style={{ color: 'black' }}
+                    key="allRegions"
+                    value="allRegions"
+                  >
+                    All
+                  </MenuItem>
+                  <ListSubheader>Administrative Regions</ListSubheader>
+                  ...
+                  {regionNames.map(region => {
+                    return (
+                      <MenuItem
+                        style={{ color: 'black' }}
+                        key={region}
+                        value={region}
+                      >
+                        {region}
+                      </MenuItem>
+                    );
+                  })}
+                  ]
+                </Select>
+              </div>
+              <div className={classes.AlertFormOptions}>
+                <TextField
+                  id="email-address"
+                  className="emailAddressClass"
+                  label="Email Address"
+                  type="text"
+                  variant="filled"
+                  onChange={onChangeEmail}
+                />
+              </div>
+            </div>
+            <Button
+              className={classes.innerAnalysisButton}
+              onClick={runAlertForm}
+              disabled={!hazardLayerId || !selectedStat || !emailValid}
+            >
+              <Typography variant="body2">Create Alert</Typography>
+            </Button>
+          </div>
+        ) : null}
+      </Box>
+    </div>
+  ) : null;
+}
+
+const styles = (theme: Theme) =>
+  createStyles({
+    AlertForm: {
+      zIndex: theme.zIndex.drawer - 1, // position this below the Analysis drawer
+      position: 'absolute',
+      top: 40,
+      left: 2,
+      textAlign: 'left',
+    },
+    AlertFormMenu: {
```

Comment:
We are switching between PascalCase and camelCase here. Maybe we stick to just camelCase? Since I think we use that everwhere.

Reply:
Sure thing; this may have been a find-and-replace mistake